composer.json

```diff
@@ -22,12 +22,12 @@
         "cweagans/composer-patches": "^1.6.5",
         "drupal-composer/drupal-scaffold": "^2.5",
         "drupal/console": "^1.0.2",
-        "drupal/core": "8.6.x-dev",
+        "drupal/core": "8.7.1",
```

Comment:
These were set to placeholder values to allow for the .env file to be definitive about a release - govcms version, drupal version, amazee version, php version
https://github.com/govCMS/govcms8lagoon/blob/master/.docker/Dockerfile.govcms8#L11-L12

but happy to discuss ;-)

Reply:
Right! Totally missed when this change happened and clearly didn't RTFM ;)

Removed pinning, should still resolve composer memory install issue + writable settings.php file (warning in site reports).